docs/howto/import.md

```diff
@@ -22,6 +22,22 @@ To avoid copying the data, lakeFS offers [Zero-copy import](#zero-copy-import).
 
 ### Prerequisites
 
+#### User Permissions
+
+The lakeFS user running the import must have the following permissions:
+`fs:WriteObject`, `fs:CreateMetaRange`, `fs:CreateCommit` and 
+`fs:ImportFromStorage`. The first 3 permissions are available by default to 
```

Comment:
```suggestion
To run import you need the following permissions:
`fs:WriteObject`, `fs:CreateMetaRange`, `fs:CreateCommit`, and 
`fs:ImportFromStorage`. The first 3 permissions are available by default to 
```

(Note the extra "Oxford comma".  I think @YaelRiv had a rule to use those, but of course we can change it.  @rmoff , any preference?